ui/packages/catalog-ui-search/src/main/webapp/component/property/property.collection.view.js

```diff
@@ -352,22 +358,26 @@ module.exports = Marionette.CollectionView.extend(
       })
     },
     determinePropertyIntersection: function(metacards) {
-      var metacardTypes = metacards.reduce((types, metacard) => {
+      const metacardTypes = metacards.reduce((types, metacard) => {
         if (types.indexOf(metacard['metacard-type']) === -1) {
           types.push(metacard['metacard-type'])
         }
         return types
       }, [])
-      var typeIntersection = _.intersection.apply(
+      const typeIntersection = _.intersection.apply(
         _,
-        metacardTypes.map(type => {
-          return Object.keys(metacardDefinitions.metacardDefinitions[type])
-        })
+        metacardTypes
+          .filter(
+            type => metacardDefinitions.metacardDefinitions[type] !== undefined
```

Comment:
❓ This is to fix issues with missing types due to their lack of whitelisting right?  What does it look like when looking at a metacard with a missing type?  Do we still show things like title?

Reply:
If we don't have the type we can't find various field definitions.